Odata-docs/client/v7/using-extensions.md

```diff
@@ -40,7 +40,7 @@ The IODataClientFactory can be registered by calling the AddODataClient extensio
 services.AddODataClient();
 ```
 
-Once registered, code can accept an IODataClientFactory anywhere services can be injected with dependency injection (DI). The IODataClientFactory can be used to create a generated client proxy instance, then follow [Basic CRUD operations](/odata/client/basic-crud-operations) to use client proxy in business logic:
+Once registered, code can accept an IODataClientFactory anywhere services can be injected with dependency injection (DI). The IODataClientFactory can be used to create a generated client proxy instance, then follow [Basic CRUD Operations](/odata/client/basic-crud-operations.md) to use client proxy in business logic:
```

Comment:
This [comment](https://github.com/MicrosoftDocs/OData-docs/pull/149/files#r505174949) applies here as well. If you check through other links you'll find that `.md` is left out

Reply:
#Resolved